GAPI.class.php

```diff
@@ -73,16 +69,7 @@ function GAPI($username, $password)
         $this->username = $username;
         $this->password = $password;
 
-        $json_handler = new Httpful\Handlers\JsonHandler(array('decode_as_array' => true));
-        Httpful\Httpful::register('application/json', $json_handler);
-
-        $template = Request::init()
-            ->addHeader('Accept', 'application/json;')
-            ->addHeader('Authorization', 'Token ' . $this->password)
-            ->expects('application/json')
-            ->sendsJson();
-
-        Request::ini($template);
+        $this->request = _wp_http_get_object();
```

Comment:
As stated in wordpress documentation, this is a private wp method and usage inside plugins is not allowed.
https://developer.wordpress.org/reference/functions/_wp_http_get_object/